### PR TITLE
Support media playback keys (song forward & song backwards)

### PR DESCRIPTION
**Pull request by mjsir911**
I had to change from handling keycodes to handling .key too to do this, which is apparently [deprecated anyways](https://stackoverflow.com/a/4471635)

### CODE REVIEW

File: src/js/HuesCore.ts [hunk 1]
```diff
@@ -1162,85 +1162,83 @@ export class HuesCore extends EventListener<CoreEvents> {
...
-        case 109: // NUMPAD_SUBTRACT
-        case 189: // MINUS
-        case 173: // MINUS, legacy
+        case "-":
             this.soundManager.decreaseVolume();
             break;
-        case 107: // NUMPAD_ADD
-        case 187: // EQUAL
-        case 61: // EQUAL, legacy
+        case "=": // actualy a "+" but shift is disabled
```
Comment (mon) [inline src/js/HuesCore.ts:1187]: Why can't we do something like `e.shiftKey`?

Comment (mjsir911) [reply src/js/HuesCore.ts:1187]: In the calling function shiftKey is intercepted:

https://github.com/mon/0x40-web/blob/694c6fd4261929b6a60d18e0ae023349f8259e6b/src/js/HuesCore.ts#L335-L337

even if it wasn't, the case would still end up with "=", the event for typing "+" would look like `{.key="=", .shiftKey=true}`

Was just trying to maintain existing behaviour, currently `=` works but `+` doesn't for volume control on https://0x40.mon.im

**Comment by mon** [comment]
Thanks for this btw, it's bothered me for a while that I was using the deprecated keycodes ^_^

**Comment by mon** [comment]
FYI: both this and #75 have shipped in 0x40-web v5.2 🎉

**Pull request merged**